core/src/main/java/org/pentaho/di/core/Const.java

```diff
@@ -754,6 +754,12 @@ public String getMessage() {
    */
   public static final String KETTLE_DO_NOT_NORMALIZE_NULL_STRING_TO_EMPTY = "KETTLE_DO_NOT_NORMALIZE_NULL_STRING_TO_EMPTY";
 
+
+  /**
+   * This flag will force to return the original string with only spaces instead of an empty string.
+   */
+  public static final String KETTLE_FORCE_STRING_WITH_SPACES_ONLY_AS_NOT_EMPTY = "KETTLE_FORCE_STRING_WITH_SPACES_ONLY_AS_NOT_EMPTY";
```

Comment:
i would call it something like the kettle property above. 
Something like:
KETTLE_DO_NOT_NORMALIZE_SPACES_ONLY_STRING_TO_EMPTY

What do you think?

Reply:
:+1: